### ISSUE TITLE
Maybe citus_total_relation_size should return NULL when the relation does not exist

### ISSUE THREAD

**Issue by lithp** [citusdata/citus#1705]
This would be consistent with `pg_table_size`, more discussion is in #1703.

**Comment by ozgune** [comment]
@lithp -- The pull request related to this issue has been merged. Can we close this issue?

**Comment by lithp** [comment]
@ozgune no, both of those PRs reference this issue but neither addresses it. [Here's the specific comment](https://github.com/citusdata/citus/pull/1703#discussion_r144495872) this issue refers to.
